dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00134</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00137</BuildToolsVersion>
```

Comment:
I'll do a quick(-ish) verification of using 141 in TFS as my change doesn't modify anything else related to the TFS build.